fineract-provider/src/main/java/org/apache/fineract/portfolio/group/api/GroupsApiResource.java

```diff
@@ -624,11 +624,11 @@ public String retrieveGsimAccounts(@PathParam("groupId") final Long groupId,
 
         }
 
-        final Set<String> GSIM_ACCOUNTS_DATA_PARAMETERS = new HashSet<>(
+        final Set<String> glimAccountsDataParameters = new HashSet<>(
                 Arrays.asList("gsimId", "groupId", "accountNumber", "childGSIMAccounts", "parentBalance", "savingsStatus"));
```

Comment:
Mistake here I think - shouldn't this be gsim not glim - as in the original code?

Reply:
updated, thanks for finding this.